resourceCreation.ps1

```diff
@@ -199,20 +215,27 @@ try {
     foreach ($resource in $resourceContext.SourceData) {
         Write-Information "Checking $($resource)"
         try {
-            # Example: Department (department differs from other objects as the property for the name is "DisplayName", not "Name")
-            $groupBody = [PSCustomObject]@{
-                schemas      = "urn:ietf:params:scim:schemas:core:2.0:Group"
-                external_id  = "department_$($resource.ExternalId)"
-                display_name = "department_$($resource.DisplayName)"
-            }
+            # Example: department_<department externalId>
+            $correlationValue = "department_" + $contract.Department.ExternalId
```

Comment:
In resource creation we don't have the $contract variable. Copy and paste error?

Reply:
Copy paste error indeed, has been fixed.